Lib/test/test_tools/test_i18n.py

```diff
@@ -155,206 +155,6 @@ def test_POT_Creation_Date(self):
             # This will raise if the date format does not exactly match.
             datetime.strptime(creationDate, '%Y-%m-%d %H:%M%z')
 
-    def test_funcdocstring(self):
-        for doc in ('"""doc"""', "r'''doc'''", "R'doc'", 'u"doc"'):
-            with self.subTest(doc):
-                msgids = self.extract_docstrings_from_str(dedent('''\
-                def foo(bar):
-                    %s
-                ''' % doc))
-                self.assertIn('doc', msgids)
-
-    def test_funcdocstring_bytes(self):
-        msgids = self.extract_docstrings_from_str(dedent('''\
-        def foo(bar):
-            b"""doc"""
-        '''))
-        self.assertFalse([msgid for msgid in msgids if 'doc' in msgid])
-
-    def test_funcdocstring_fstring(self):
-        msgids = self.extract_docstrings_from_str(dedent('''\
-        def foo(bar):
-            f"""doc"""
-        '''))
-        self.assertFalse([msgid for msgid in msgids if 'doc' in msgid])
-
-    def test_classdocstring(self):
-        for doc in ('"""doc"""', "r'''doc'''", "R'doc'", 'u"doc"'):
-            with self.subTest(doc):
-                msgids = self.extract_docstrings_from_str(dedent('''\
-                class C:
-                    %s
-                ''' % doc))
-                self.assertIn('doc', msgids)
-
-    def test_classdocstring_bytes(self):
-        msgids = self.extract_docstrings_from_str(dedent('''\
-        class C:
-            b"""doc"""
-        '''))
-        self.assertFalse([msgid for msgid in msgids if 'doc' in msgid])
-
-    def test_classdocstring_fstring(self):
-        msgids = self.extract_docstrings_from_str(dedent('''\
-        class C:
-            f"""doc"""
-        '''))
-        self.assertFalse([msgid for msgid in msgids if 'doc' in msgid])
-
-    def test_moduledocstring(self):
-        for doc in ('"""doc"""', "r'''doc'''", "R'doc'", 'u"doc"'):
-            with self.subTest(doc):
-                msgids = self.extract_docstrings_from_str(dedent('''\
-                %s
-                ''' % doc))
-                self.assertIn('doc', msgids)
-
-    def test_moduledocstring_bytes(self):
-        msgids = self.extract_docstrings_from_str(dedent('''\
-        b"""doc"""
-        '''))
-        self.assertFalse([msgid for msgid in msgids if 'doc' in msgid])
-
-    def test_moduledocstring_fstring(self):
-        msgids = self.extract_docstrings_from_str(dedent('''\
-        f"""doc"""
-        '''))
-        self.assertFalse([msgid for msgid in msgids if 'doc' in msgid])
-
-    def test_msgid(self):
-        msgids = self.extract_docstrings_from_str(
-                '''_("""doc""" r'str' u"ing")''')
-        self.assertIn('docstring', msgids)
-
-    def test_msgid_bytes(self):
```

Comment:
This is not tested by the snapshot tests, you can either leave it here, or move it inside i18n_data/messages.py  under `Invalid arguments`